### PR TITLE
Created application.rst in portuguese

### PR DESCRIPTION
**Pull request by tavaresgerson**
Added new file in _development/application.rst_, exist in folder "en/", but don't "pt/"

### CODE REVIEW

File: pt/development/errors.rst [hunk 1]
```diff
@@ -1,17 +1,610 @@
...
+Configurações de Erro & Exceções
+================================
+
+A configuração do erro é feita no arquivo **config/app.php** do seu aplicativo. Por padrão, o CakePHP usa 
+``Cake\Error\ErrorHandler`` para lidar com erros e exceções do PHP por padrão. A configuração de erro permite 
+personalizar o tratamento de erros para o seu aplicativo. As seguintes opções são suportadas:
+
+* ``errorLevel`` - int - O nível de erros que você está interessado em capturar. Use as constantes de erro embutidas no PHP e máscaras de bits para selecionar o nível de erro no qual você precisa. Você pode configurá-lo como ``E_ALL ^ E_USER_DEPRECATED`` para desativar os avisos de depreciação.
+* ``trace`` - bool - Inclua rastreamentos de pilha para erros nos arquivos de log. Rastreamentos de pilha serão incluídos no log após cada erro. Isso é útil para descobrir onde/quando os erros estão sendo gerados
+* ``exceptionRenderer`` - string - A classe responsável por renderizar exceções não capturadas. Se você escolher uma classe personalizada, coloque o arquivo dessa classe em ** src/Error**. Esta classe precisa implementar o método ``render()``.
```
Comment (markstory) [inline pt/development/errors.rst]:
```suggestion
* ``exceptionRenderer`` - string - A classe responsável por renderizar exceções não capturadas. Se você escolher uma classe personalizada, coloque o arquivo dessa classe em **src/Error**. Esta classe precisa implementar o método ``render()``.
```

File: pt/development/errors.rst [hunk 2]
```diff
@@ -1,17 +1,610 @@
...
+.. php:class:: ExceptionRenderer(Exception $exception)
+
+Changing Exception Handling
+===========================
+
+Exception handling offers several ways to tailor how exceptions are handled.  Each
+approach gives you different amounts of control over the exception handling
+process.
+
+#. *Customize the error templates* This allows you to change the rendered view
```
Comment (markstory) [inline pt/development/errors.rst]:
```suggestion
#. **Customize the error templates** This allows you to change the rendered view
```

File: pt/development/errors.rst [hunk 3]
```diff
@@ -1,17 +1,610 @@
...
+   templates as you would any other template in your application.
+#. *Customize the ErrorController* This allows you to control how exception
```
Comment (markstory) [inline pt/development/errors.rst]:
```suggestion
#. **Customize the ErrorController** This allows you to control how exception
```

File: pt/development/errors.rst [hunk 4]
```diff
@@ -1,17 +1,610 @@
...
+   pages are rendered.
+#. *Customize the ExceptionRenderer* This allows you to control how exception
```
Comment (markstory) [inline pt/development/errors.rst]:
```suggestion
#. **Customize the ExceptionRenderer** This allows you to control how exception
```

File: pt/development/errors.rst [hunk 5]
```diff
@@ -1,17 +1,603 @@
...
+Configurações de Erro & Exceções
+================================
+
+A configuração do erro é feita no arquivo **config/app.php** do seu aplicativo. Por padrão, o CakePHP usa 
+``Cake\Error\ErrorHandler`` para lidar com erros e exceções do PHP por padrão. A configuração de erro permite 
+personalizar o tratamento de erros para o seu aplicativo. As seguintes opções são suportadas:
+
+* ``errorLevel`` - int - O nível de erros que você está interessado em capturar. Use as constantes de erro embutidas no PHP e máscaras de bits para selecionar o nível de erro no qual você precisa. Você pode configurá-lo como ``E_ALL ^ E_USER_DEPRECATED`` para desativar os avisos de depreciação.
+* ``trace`` - bool - Inclua rastreamentos de pilha para erros nos arquivos de log. Rastreamentos de pilha serão incluídos no log após cada erro. Isso é útil para descobrir onde/quando os erros estão sendo gerados
+* ``exceptionRenderer`` - string - A classe responsável por renderizar exceções não capturadas. Se você escolher uma classe personalizada, coloque o arquivo dessa classe em ** src/Error**. Esta classe precisa implementar o método ``render()``.
```
Comment (markstory) [inline pt/development/errors.rst]:
```suggestion
* ``exceptionRenderer`` - string - A classe responsável por renderizar exceções não capturadas. Se você escolher uma classe personalizada, coloque o arquivo dessa classe em **src/Error**. Esta classe precisa implementar o método ``render()``.
```

**Comment by othercorey** [comment]
@tavaresgerson Are you able to make the last fix so we can commit this? Are there any other issues?

**Comment by tavaresgerson** [comment]
> @tavaresgerson Are you able to make the last fix so we can commit this? Are there any other issues?

All done!

**Pull request merged**